ceph.spec.in

```diff
@@ -2151,6 +2156,48 @@ fi
 %{_bindir}/crimson-osd
 %endif
 
+%files volume
+%{_sbindir}/ceph-volume
+%{_sbindir}/ceph-volume-systemd
+%dir %{python3_sitelib}/ceph_volume
+%{python3_sitelib}/ceph_volume/*
+%{python3_sitelib}/ceph_volume-*
+%{_mandir}/man8/ceph-volume.8*
+%{_mandir}/man8/ceph-volume-systemd.8*
+%{_unitdir}/ceph-volume@.service
+
+%post volume
```

Comment:
shouldn't we also remove ceph-volume from patch : https://github.com/ceph/ceph/blob/f8f7b865715987139d96e4baf41c82329dc19108/ceph.spec.in#L2127-L2147

Reply:
thanks. fixed.